src/client/components/PayerBalance.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+
+import translate from '~/common/services/i18n';
+import web3 from '~/common/services/web3';
+import BoxRounded from '~/client/components/BoxRounded';
+
+const PayerBalance = () => {
+  const [balance, setBalance] = useState(0);
+
+  useEffect(() => {
+    const getBalance = async () => {
+      const state = await web3.eth.getBalance(process.env.PAYER_ADDRESS);
+      setBalance(state);
+    };
+    getBalance();
+  }, [setBalance]);
+
+  return (
+    <main>
```

Comment:
`main` would suggest semantically the main content of the website (except of if you wrap it around an `article` etc.). I would suggest to just remove it and use `BoxRounded` as the root.